spec.md

```diff
@@ -540,7 +615,7 @@ response has in their list of known nodes.
 
 ### Packed node format
 
-The DHT Send nodes uses the Packed Node Format.
+The DHT Send Nodes uses Packed Node Format.
```

Comment:
Ugh, not sure; It was late night... Maybe I felt like it's kind of name so no need for?